### PR TITLE
Add join table for visits

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Understood the different types of relationships between tables.
- Prepared complex queries that answer analytical questions.
- Used primary key & foreign key mechanism for joining tables.

Screenshot (105)
Screenshot (106)
Screenshot (107)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -91,4 +91,60 @@ WHERE full_name = 'Dean Winchester' AND escape_attempts = 0;
 -- Who owns the most animals?
 SELECT COUNT(*) AS owners_count, owners.full_name
 FROM animals JOIN owners ON owners.id = animals.owners_id
-GROUP BY owners.full_name ORDER BY owners_count DESC LIMIT 1;
\ No newline at end of file
+GROUP BY owners.full_name ORDER BY owners_count DESC LIMIT 1;
+
+/* Write queries to answer questions */
```
Comment (danmainah) [inline queries.sql:96]: - [ ] @shyusu4 great work on your queries however you forgot to add a screenshot of your queries in the pull request description, this enables checking that your queries run successfully and that it's a project requirement.
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shyusu4 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (shyusu4) [reply queries.sql:96]: Hi! Thanks for the review. I'll add the screenshots.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**